secator/runners/_helpers.py

```
@@ -4,13 +4,14 @@
 from secator.utils import deduplicate, debug
 
 
-def run_extractors(results, opts, inputs=[], dry_run=False):
+def run_extractors(results, opts, inputs=[], ctx={}, dry_run=False):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)